### PR TITLE
Overhaul the readme

### PR DESCRIPTION
**Pull request by lgierth**

### CODE REVIEW

**Comment by lgierth** [comment]
Converted the packages links to a table

**Comment by lgierth** [comment]
This is good to go now. Fixed a few links and added unixfs to the table.

In the next pass I'll add some info on the state of IPLD in go-ipfs.

**Comment by lgierth** [comment]
Oh and brief case studies would be neat too (search index for wikipedia-on-ipfs.org, eth-ipfs bridge)

File: README.md [hunk 1]
```diff
@@ -4,95 +4,53 @@
...
-[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-ipld-blue.svg?style=flat-square)](http://github.com/ipld/ipld)
-[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai)
+[![](https://img.shields.io/badge/project-ipld-blue.svg?style=flat-square)](https://github.com/ipld/ipld)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](https://webchat.freenode.net/?channels=%23ipfs)
 
 > InterPlanetary Linked Data
 
-Read more about IPLD at https://ipld.io
```
Comment (Stebalien) [inline README.md]: Any real reason to get rid of this link? ipld.io is live and well.

File: README.md [hunk 2]
```diff
@@ -4,95 +4,53 @@
...
-- [go-ipld-node](https://github.com/ipfs/go-ipld-node)
-
-#### Interface definitions
 
-- [interface-ipld-format](https://github.com/ipld/interface-ipld-format) - A interface you can follow to implement a valid IPLD format, resolvable through the IPLD Resolver (available in IPFS)
+## Overview
+
+- Specs: [ipld/specs](https://github.com/ipld/specs)
+- CID specs: [ipld/cid](https://github.com/ipld/cid)
+- Website: [ipld/website](https://github.com/ipld/website)
```
Comment (Stebalien) [inline README.md]: Not something I'd put here (kind of an internal thing).

File: README.md [hunk 3]
```diff
@@ -4,95 +4,53 @@
...
-- [go-ipld-zcash](https://github.com/ipfs/go-ipld-zcash)
-- [go-ipld-cbor](https://github.com/ipfs/go-ipld-cbor)
-- [go-ipld-git](https://github.com/ipfs/go-ipld-git)
-- [go-ipld-node](https://github.com/ipfs/go-ipld-node)
-
-#### Interface definitions
 
-- [interface-ipld-format](https://github.com/ipld/interface-ipld-format) - A interface you can follow to implement a valid IPLD format, resolvable through the IPLD Resolver (available in IPFS)
+## Overview
+
```
Comment (Stebalien) [inline README.md]: We should probably add a blurb here explaining *WTF* is IPLD. Honestly, we can just steal it from the website:

> IPLD is the data model of the content-addressable web. It allows us to treat all hash-linked data structures as subsets of a unified information space, unifying all data models that link data with hashes as instances of IPLD.

File: README.md [hunk 4]
```diff
@@ -4,95 +4,53 @@
...
-
-- [Repositories](#repositories)
-  - [Specifications](#specifications)
-  - [Implementations](#implementations)
-    - [New implementations](#new-implementations)
-  - [Other Tools](#other-tools)
-  - [Meta repos](#meta-repos)
-- [Lead](#lead)
+- [Overview](#overview)
+- [New implementations](#new-implementations)
```
Comment (Stebalien) [inline README.md]: There's a bullet point about this but no section.

File: README.md [hunk 5]
```diff
@@ -4,95 +4,53 @@
...
+- Website: [ipld/website](https://github.com/ipld/website)
+- Tools
+  - IPLD graph builder: [ipld/js-ipld-graph-builder](https://github.com/ipld/js-ipld-graph-builder)
+  - CLI for interacting with IPLD: [ipld/js-ipld-cli](https://github.com/ipld/js-ipld-cli)
+- New implementations
+  - Are you working on your own implementation in another language?
+  - [Open an issue](https://github.com/ipld/ipld/issues) in this repository to discuss it with others, find help, and coordinate efforts.
+  - Eventually, we can move it to the organization if you like, add it above, and mention it on the website.
+
+
```
Comment (Stebalien) [inline README.md]: New section? (Implementations)

File: README.md [hunk 6]
```diff
@@ -4,95 +4,53 @@
...
-### IPLD Definitions
-
-There are a variety of systems that use merkle-tree and hash-chain inspired datastructures (e.g. git, bittorrent, IPFS, tahoe-lafs, sfsro). IPLD defines:
-
-- merkle-links: the core unit of a merkle-graph
-- merkle-dag: any graphs whose edges are merkle-links.
-- merkle-paths: unix-style paths for traversing merkle-dags with named merkle-links
-- IPLD Data Model: a flexible, JSON-based data model for representing merkle-dags.
-- IPLD Serialized Formats: a set of formats in which IPLD objects can be represented, for example JSON, CBOR, CSON, YAML, Protobuf, XML, RDF, etc.
-- IPLD Canonical Format: a deterministic description on a serialized format that ensures the same logical object is always serialized to the exact same sequence of bits. This is critical for merkle-linking, and all cryptographic applications.
```
Comment (Stebalien) [inline README.md]: So, this information can actually be quite useful. Personally, I'd get rid of the last point (the concept "canonical format" is a bit in flux) and change "a flexible, JSON-based data model for representing merkle-dags" to "a flexible, JSON-inspired, self-describing structured data model for representing merkle-dags." (at the risk of being a bit wordy).

Probably in a section at the bottom. I believe this list grew out of a lot of repeated confusion/questions so getting rid of it wholesale may not be the best idea.

Comment (lgierth) [reply README.md]: I opted to remove it while the spec work is still ongoing, to have this place be mostly centered around pointing directions. Are all these terms accurate?

(Agreed about all other comments.)

**Comment by lgierth** [comment]
> Also, a link to one of @diasdavid's JS-IPLD tutorials may be useful. Dealing with IPLD from javascript is actually reasonably pleasant.

Where are they?

File: README.md (hunk 6 above)
Comment (Stebalien) [reply README.md]: Except for the pieces I pointed out, yes. We still talk about merkle-links, merkle-dags, merkle-paths, and serialization formats. Personally, I'd keep this and frame it as an FYI. That is, "As you read through documents related to IPLD, you may come across the following terms:".

**Comment by Stebalien** [comment]
> Where are they?

Here's a good one: https://github.com/ipfs/js-ipfs/tree/master/examples/traverse-ipld-graphs

File: README.md (hunk 2 above)
Comment (lgierth) [reply README.md]: I disagree, these repos are not internal. I moved them down below the packages table though.

**Comment by lgierth** [comment]
I addresses your feedback, and added a few more examples (git-remote-ipld, git in go-ipfs, wikipedia search).
